### DISCUSSION TITLE
How to add specific project configuration

### DISCUSSION THREAD

**Discussion by mikaoelitiana** [LazyVim/LazyVim#139] [Q&A]
Is there a way to add specific configurations for a project (`/path/to/project/init.lua` for example)?

**Comment by luxus** [comment]
why do need that? u can load plugins via file extension and direnv will help you with dev env. for binaries

**Reply by mikaoelitiana** [reply]
My current use case is setting [vim-test](https://github.com/vim-test/vim-test/#javascript) variables. On some projects, the test runner is Jest and on other projects we have Mocha. I want to set `let g:test#javascript#runner = 'jest'` accordingly. 
I did not know direnv, I will take a look into it.

**Reply by folke** [reply]
You could probably use `:h exrc` for that if you're on nightly.

**Reply by mikaoelitiana** [reply]
Thanks, I will check that
